test-lib.sh

```diff
@@ -518,6 +558,10 @@ EOF
     fi
     # Filter out env var definitions from $s2i_args and create Dockerfile ENV commands out of them
     echo "$s2i_args" | grep -o -e '\(-e\|--env\)[[:space:]=]\S*=\S*' | sed -e 's/-e /ENV /' -e 's/--env[ =]/ENV /' >>"$df_name"
+    mount_ca_option=$(echo "$s2i_args" | grep -o -e '\(-v\)[[:space:]]/etc/pki/\.*\S*')
+    if [ x"$mount_ca_option" != "x" ]; then
+      echo "RUN cd /etc/pki/ca-trust/source/anchors && update-ca-trust extract" >>"$df_name"
+    fi
```

Comment:
Can this be moved to assembly script and run unconditionally?

Reply:
This has  not to be run in `assemble` script, because it is run as `USER 1001`